bluemira/utilities/tools.py

```diff
@@ -72,10 +72,6 @@ def floatstr(_floatstr, obj, *args, **kwargs):
         https://bugs.python.org/issue42434
         https://bugs.python.org/issue31466
         """
-        from bluemira.base.parameter import Parameter
```

Comment:
I assume our new fancy Parameters can do this automatically?

Reply:
Do you know what the original intent of this is? If I try to encode a `Parameter` in the JSON, I get an error, even with this in

Reply:
I had to look it up. It was because a Parameter used to identify as the type it was wrapping so the floatstr conversion needed modifying because json writer used `__repr__` to do type checking instead of `isinstance`. Now the whole function could probably be removed. Also while I remember we can remove `wrapt` as a direct dependency now

Reply:
Ah, that makes sense. I've removed it - let's see what breaks 😄